storybookjs/storybook
# Vue: Add vue3 support

Issue: #10654 

## What I did
Added Vue 3 support to @storybook/vue

Thanks to @andoshin11
https://www.npmjs.com/package/@andoshin11/storybook-vue3

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

We should make another example like kitchen sink with Vue 3
